Description: Grab translatable strings from debian/* files
 This is done as an Ubuntu patch, since debian/* files are not included upstream.
Forwarded: not-needed
Author: Gunnar Hjalmarsson <gunnarhj@ubuntu.com>

--- lightdm.orig/po/POTFILES.in	2014-09-30 05:00:39.000000000 +0200
+++ lightdm/po/POTFILES.in	2014-10-19 18:08:09.576134427 +0200
@@ -1,6 +1,9 @@
 # List of source files containing translatable strings.
 # Please keep this file sorted alphabetically.
 [encoding: UTF-8]
+debian/config-error-dialog.sh
+debian/guest-account.sh
+debian/guest-session-auto.sh
 data/org.freedesktop.DisplayManager.AccountsService.policy.in
 src/lightdm.c
 src/dm-tool.c
